tests/gis_tests/gdal_tests/test_ds.py

```diff
@@ -9,17 +10,26 @@
 
 from ..test_data import TEST_DATA, TestDS, get_ds_file
 
+wgs_84_wkt = (
+    'GEOGCS["GCS_WGS_1984",DATUM["WGS_1984",SPHEROID["WGS_1984",'
+    '6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",'
+    '0.017453292519943295]]'
+)
+# Using a regex because of small differences depending on GDAL versions.
+# AUTHORITY part has been added in GDAL 2.2.
+wgs_84_wkt_regex = (
+    r'^GEOGCS\["GCS_WGS_1984",DATUM\["WGS_1984",SPHEROID\["WGS_1984",'
```

Comment:
It seems that the third "WGS_1984" on this line is normalized to "WGS_84". With that change, this is passing for me on ubuntu 14.04/16.04.